From 78f71c3b83f64a6966a1ff5e96ffda00be02a760 Mon Sep 17 00:00:00 2001
From: Atul Dhudase <quic_adhudase@quicinc.com>
Date: Tue, 31 Oct 2023 11:18:40 +0530
Subject: [PATCH 25/27] PENDING: dt-bindings: pinctrl: qcom,sc7280-pinctrl: add
 gpio-reserved-ranges

Add gpio-reserved-ranges property for SC7280 (used on QCM6490 boards).

Change-Id: Ifd1943314dc13170aeef31a4d9bdf137b61d49ce
Signed-off-by: Atul Dhudase <quic_adhudase@quicinc.com>
Signed-off-by: Atul Dhudase <quic_adhudase@quicinc.com>
Upstream-Status: Pending
---
 .../devicetree/bindings/pinctrl/qcom,sc7280-pinctrl.yaml      | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/pinctrl/qcom,sc7280-pinctrl.yaml b/Documentation/devicetree/bindings/pinctrl/qcom,sc7280-pinctrl.yaml
index 368d44ff5468..c8735ab97e40 100644
--- a/Documentation/devicetree/bindings/pinctrl/qcom,sc7280-pinctrl.yaml
+++ b/Documentation/devicetree/bindings/pinctrl/qcom,sc7280-pinctrl.yaml
@@ -41,6 +41,10 @@ properties:
   gpio-ranges:
     maxItems: 1
 
+  gpio-reserved-ranges:
+    minItems: 1
+    maxItems: 88
+
   gpio-line-names:
     maxItems: 175
 
-- 
2.25.1

